src/KK_Hooks_KK/HOutfitFolders.cs

```diff
@@ -0,0 +1,125 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Reflection;
+using System.Reflection.Emit;
+using BepInEx.Harmony;
+using ChaCustom;
+using HarmonyLib;
+using KKAPI.Utilities;
+using Manager;
+using UnityEngine;
+using UnityEngine.UI;
+
+namespace BrowserFolders.Hooks.KK
+{
+    [BrowserType(BrowserType.HOutfit)]
+    public class HOutfitFolders : IFolderBrowser
+    {
+        private static clothesFileControl _customCoordinateFile;
+        private static FolderTreeView _folderTreeView;
+
+        private static string _currentRelativeFolder;
+        private static bool _refreshList;
+
+        public HOutfitFolders()
+        {
+            _folderTreeView = new FolderTreeView(Utils.NormalizePath(UserData.Path), Utils.NormalizePath(UserData.Path));
+            _folderTreeView.CurrentFolderChanged = OnFolderChanged;
+
+            Harmony.CreateAndPatchAll(typeof(HOutfitFolders));
+        }
+
+        private static string DirectoryPathModifier(string currentDirectoryPath)
+        {
+            return _folderTreeView != null ? _folderTreeView.CurrentFolder : currentDirectoryPath;
+        }
+
+        [HarmonyPrefix]
+        [HarmonyPatch(typeof(clothesFileControl), "Start")]
+        internal static void InitHook(clothesFileControl __instance)
+        {
+            _folderTreeView.DefaultPath = Path.Combine((UserData.Path), "coordinate/");
+            _folderTreeView.CurrentFolder = _folderTreeView.DefaultPath;
+
+            _customCoordinateFile = __instance;
+        }
+
+        [HarmonyTranspiler]
+        [HarmonyPatch(typeof(clothesFileControl), "Initialize")]
+        internal static IEnumerable<CodeInstruction> InitializePatch(IEnumerable<CodeInstruction> instructions)
+        {
+            foreach (var instruction in instructions)
+            {
+                if (string.Equals(instruction.operand as string, "coordinate/", StringComparison.OrdinalIgnoreCase))
+                {
+                    //0x7E	ldsfld <field>	Push the value of the static field on the stack.
+                    instruction.opcode = OpCodes.Ldsfld;
+                    instruction.operand = typeof(HOutfitFolders).GetField(nameof(_currentRelativeFolder), BindingFlags.NonPublic | BindingFlags.Static);
+                }
+
+                yield return instruction;
+            }
+        }
+
+        public void OnGui()
+        {
+            var guiShown = false;
+            
+            if (GameObject.Find("Canvas/clothesFileWindow").activeSelf) //if preset window is active draw file select
+            {
+                if (_refreshList)
```

Comment:
_refreshList doesn't seem to be set anywhere so this if will not do anything. It was used to update the list when user saved a new coordinate in maker